routes/base.py

```diff
@@ -150,6 +150,27 @@ def format_meta_title(title, locale):
         translate('defaultTitle', locale)
     )
 
+def get_custom_parameters():
```

Comment:
@Roblinde In python the convention is to avoid names starting with `get_`, please change it to something like `parameterized_url` or something of that sort.

Cheers

Reply:
will do